### ISSUE TITLE
XLABS sampler is slower than ksamper

### ISSUE THREAD

**Issue by starinskycc** [XLabs-AI/x-flux-comfyui#100]
I tested it in Comfyui, XLABS sampler is slower than ksamper

**Comment by starinskycc** [comment]
![image](https://github.com/user-attachments/assets/cd79bde0-3c51-4be6-9bcd-d5956f139b07)
![1](https://github.com/user-attachments/assets/f6099b56-9d7c-404e-9126-354ff76fb9e6)

**Comment by LuckyNES** [comment]
I agree. It is much slower. I would like to be able to use the controlnets with KSampler if possible or for XSampler to expose the choice to choose sampler_name like KSampler has. Also the results I get from XSampler are much WORSE than if I use deis in KSampler.

**Comment by F0xbite** [comment]
I've noticed this too. It's about 2x slower than standard Ksampler. 

It's definitely the sampler and not controlnet, because using the xlabs sampler without controlnet still takes 2x as long.

**Comment by valaxy** [comment]
agree, I thought it's my PC's problem, finally find Xlabs Sampler's

**Comment by ewebgh33** [comment]
I can't find a way to get the same result out of Xlabs Sampler vs KSampler, or Xlabs Sampler vs SamplerCustomAdvaced.
WHY is Xlabs sampler even a thing?
There is an established workflow for using controlnets, with either KSampler or SamplerCustomAdvanced. Why now introduce yet another (inferior) node?

**Comment by enesaltinkaya** [comment]
When I set **timestep_to_start_cfg**  equals or more than the **steps**, it runs as fast as ksampler.  
  
You know how we set cfg to 1 when using flux with ksampler, timestep_to_start_cfg must be setting it more than 1, so it slows down.

**Comment by Stardesign2024** [comment]
> When I set **timestep_to_start_cfg** equals or more than the **steps**, it runs as fast as ksampler.
> 
> You know how we set cfg to 1 when using flux with ksampler, timestep_to_start_cfg must be setting it more than 1, so it slows down.

Thanks!
